test/test_chunking.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+import pytest
+
+from RAiDER.delayFcns import chunk,makeChunksFromInds,makeChunkStartInds
+
+# The purpose of these tests is to verify the chunking done on the query points during is
+# done correctly for various dimensions
+
+def checkChunks(chunk1, chunk2):
+    check = []
+    for k1,L in enumerate(chunk2):
+        for k2,arr in enumerate(L):
+            if np.allclose(arr, chunk1[k1][k2]):
+                check.append(True)
+            else:
+                check.append(False)
+    return all(check)
+
+def test_makeChunkStartInds1D():
+    chunkStartInds = [(0,), (1,), (2,), (3,), (4,)]
+    assert chunkStartInds == makeChunkStartInds((1,),(5,))
```

Comment:
No need for intermediate variable.

I'm not sure whether it's best to do `assert desired == function()` or `assert function() == desired`, but it's definitely important to be consistent, otherwise reading test output can get confusing. I have adopted the convention of always putting the desired output on the right, mostly because it is more intuitive when you are writing the test. 

Since, the other tests I've written already follow this convention, I'd say let's follow it here too. If you have a really good reason for doing it the other way, then you would need to change it for all of the tests in the project. Consistency is key.

```suggestion
    assert makeChunkStartInds((1,),(5,)) == [(0,), (1,), (2,), (3,), (4,)]
```